tests/utils/browser.js

```diff
@@ -0,0 +1,27 @@
+const fs = require('fs');
+const feedBackDocs = async (testName = 'allLogs') => {
+  const script = `(async () => {
+  const allDbList =  await indexedDB.databases();
+  const metaDbList = allDbList.filter(db => db.name.includes('pouch_medic-user') && db.name.endsWith('-meta'))
+  const feedBack = []
+  for (const idx in metaDbList){
+    const db = metaDbList[idx]
+    const nameStripped = db.name.replace('_pouch_','');
+    const metaDb = new PouchDB(nameStripped);
+    const docs = await metaDb.allDocs({ include_docs:true });
+    const feedBackDocs = docs.rows.filter(x => x.doc.type === 'feedback')
+    feedBack.push(feedBackDocs)
+  }
+  return feedBack.flat();
+})();`;
+  const feedBackDocs = await browser.executeScript(script);
```

Comment:
Because it's global I'm having trouble finding the API for the `browser` object here. If it's a WDIO browser object then it would be better to use the [executeAsync function](https://webdriver.io/docs/api/browser/executeAsync). This would mean you can pass a function rather than a string for syntax highlighting, linting, etc. If that doesn't work for some reason, then it would be better to define the function in code, and use `fn.toString()` (IIRC) to stringify it to pass to this function.

Reply:
@garethbowen  Can you have a look now. I'm passing the function to the executeAsync now instead of a string.